server/src/main/kotlin/suwayomi/tachidesk/server/ServerSetup.kt

```diff
@@ -538,4 +540,15 @@ fun applicationSetup() {
             onError = { it?.printStackTrace() },
         )
     }
+
+    Runtime.getRuntime().addShutdownHook(
+        thread(start = false) {
+            runBlocking {
+                val logger = KotlinLogging.logger("KCEF")
+                logger.debug { "Shutting down KCEF" }
+                KCEF.dispose()
```

Comment:
Is there a disposeBlocking version of this function that we can use instead of wrapping this in runBlocking?

Reply:
There is, but it just wraps the call in `runBlocking` itself:

https://github.com/DatL4g/KCEF/blob/0665269b7d6a91b0ee187f4432bb5be5ca41a112/kcef/src/main/kotlin/dev/datlag/kcef/KCEF.kt#L294-L297

Reply:
Let's use that. Less code on our end.